README.md

```diff
@@ -28,11 +28,14 @@ following steps:
      "name": "My entity name",
      "url": "https://my.entity/url",
      "email": "my@entity.org",
-     "keybase": "myentity",
-     "twitter": "myentity"
+     "keybase": "keybase_handle",
+     "twitter": "twitter_handle"
```

Comment:
actually still a bit ambiguous. should this be `@twitter_handle` or `twitter_handle` :) but i think it's fine accepting both and clients can check this.

Reply:
nevermint, i see `@` is not allowed, so all good

Reply:
So the spec is non-ambiguous as it specifies a regular expression to match.